backend-tests/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
```

Comment:
@alfrunes: Well well well. It's *this* change that causes the test to fail!

Can we leave it out, or do you need it?